astronomy/standard_product_3.hpp

```diff
@@ -80,13 +93,22 @@ class StandardProduct3 {
   // (that order is the same in the satellite ID records and within each epoch).
   std::vector<SatelliteIdentifier> const& satellites() const;
 
-  DiscreteTrajectory<ITRS> const& orbit(SatelliteIdentifier const& id) const;
+  // Each orbit may consists of several arcs, separated by missing data.
```

Comment:
"consist"

Also mention that the trajectories are in time order (hopefully).

Reply:
Done.